src/utilities/bcl/templates/ReportingMeasure/measure.rb

```diff
@@ -131,22 +134,43 @@ def run(runner, user_arguments)
       end
     end
 
-    # only try to get the annual timeseries if an annual simulation was run
-    if ann_env_pd
+    # if requested look through time series results
+    if report_drybulb_temp
+
+      # only try to get the annual timeseries if an annual simulation was run
+      if ann_env_pd
+
+        # get desired variable
+        key_value = 'Environment'
+        time_step = 'Hourly' # "Zone Timestep", "Hourly", "HVAC System Timestep"
+        variable_name = 'Site Outdoor Air Drybulb Temperature'
+        output_timeseries = sql_file.timeSeries(ann_env_pd, time_step, variable_name, key_value) # key value would go at the end if we used it.
+
+        if output_timeseries.empty?
+          runner.registerWarning('Timeseries not found.')
+        else
+          runner.registerInfo('Found timeseries.')
+
+          output_timeseries = output_timeseries.get.values
+          min = nil
+          max = nil
+          # loop through timeseries and move the data from an OpenStudio timeseries to a normal Ruby array (vector)
+          for i in 0..(output_timeseries.size - 1)
+            if min.nil? || output_timeseries[i] < min
+              min = output_timeseries[i]
+            end
+            if max.nil? || output_timeseries[i] > max
+              max = output_timeseries[i]
+            end
+          end
```

Comment:
I made the Vector enumerable in ruby in https://github.com/NREL/OpenStudio/pull/3989, so you can do `output_timeseries.each` now. Also `.min` and `.max` exists.


```ruby
[1] bcl(main)> os_vec = OpenStudio::createVector([0,1,2,3,4,5,6,7,8,9])
=> #<OpenStudio::Vector:0x00005647c0179ba8 @__swigtype__="_p_openstudio__Vector">

[2] bcl(main)> os_vec.max
=> 9.0

[3] bcl(main)> os_vec.each {|x| puts x }
0.0
1.0
2.0
3.0
4.0
5.0
6.0
7.0
8.0
9.0

[4] bcl(main)> os_vec.minmax
=> [0.0, 9.0]
```